install-usercss/install-usercss.css

```diff
@@ -37,28 +38,21 @@ input:disabled + span {
 .container {
   display: flex;
   height: 100vh;
-  align-items: stretch;
 }
 
-.main {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-}
-
-.header,
+#header,
 .warnings {
   flex: 0 0 280px;
   box-sizing: border-box;
-  padding: 15px;
+  padding: 1rem;
```

Comment:
We're using 15px in other files.

Reply:
I assume from the CSS that it's a header replacement for warnings on invalid usercss code. Can anyone check if it's accounted for in the compact layout? Guess you'd need to force an error?

Reply:
Checked out warnings. Should be good now.